### PR TITLE
milestone-one / desktop-version

### PR DESCRIPTION
**Pull request by alexander16108**
- add linter dependencies
- add HTML and CSS files
- check for linter errors
- create a PR 
- awaiting CR

### CODE REVIEW

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
Hello @alexander16108  

#### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:blush::blush:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] You need to make sure that your desktop version looks like the one in the Figma  [Template.](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) 

#### Here is what the project looks like now

wrong3
wrong2
wrong1


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

File: index.html [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+    <nav class="hamburger-menu">
+      <a href="#"><i class="fa fa-bars"></i></a>
+    </nav>
+        <ul class="nav-links hidden">
+          <li><a href="portfolio">portfolio</a></li>
+          <li><a href="About">About</a></li>
+          <li><a href="Contact">Contact</a></li>
+          <li class="envelope"><i class="fas fa-envelope-square"></i></li>
+        </ul>
+    </nav>
```
Comment (rloterh) [inline index.html]: - [ ] Your layout looks great! We however want our design to look like the design template provided in the Figma as much as possible. We actually want our design to have appropriate margins at the **top, bottom, left, and or right** proportionally. 

Here is your design:

shot1

You can see we don't have enough margin around the logo area and the left margin.

Check out the Figma design template :
shot2

You can see that we have enough margin flow around the top area. Let's unsure to add appropriate margins at the top and left side.

**Review by rloterh — CHANGES_REQUESTED** [review]
# Changes Required! ♻️

Hi @alexander16108 👋 👋, I'm Robert, your 2nd reviewer for this PR.

I must commend you for the nice work done👏.  You nailed almost all of the specifications of this project right. You did a great job by implementing most changes recommended by the previous reviewer(s). Kindly check the below comments for the necessary changes we can implement to make our application more robust.

## Good points 👍
No Linters errors ✔️
Good use of HTML semantics ✔️


## Aspects to improve ♻️

_Check the comment(s) under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider:**

- [OPTIONAL] The **Author** section of our ReadMe lacks enough info about the author(s). Let's improve this aspect by adding additional profile links such as LinkedIn, Twitter, and Github, and ensure that the highlighted profile has links that navigate to the corresponding profile URL when clicked.


Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+      <a href="#" class="my-logo">Alexander</a>
+    <nav class="hamburger-menu">
+      <a href="#"><i class="fa fa-bars"></i></a>
+    </nav>
+        <ul class="nav-links hidden right-nav">
+          <li><a href="portfolio">portfolio</a></li>
+          <li><a href="About">About</a></li>
+          <li><a href="Contact">Contact</a></li>
+          <li class="envelope"><i class="fas fa-envelope-square"></i></li>
+        </ul>
```
Comment (ElijahTobs) [inline index.html]: - [ ] _[Optional]_ The font-weight of the logo should be the same as that of the nav items.
- [ ] _[Optional]_ You should capitalize the text `portfolio`.

File: README.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ Additional description about the project and its features.
 Html + Css
 
 # Live Demo Of The Project
-No live demo.
+https://alexander16108.github.io/destop-version/
```
Comment (ElijahTobs) [inline README.md:14]: - [ ] _[Optional]_ The live demo link isn't working. It takes forever to load and eventually does not come up with anything
- [ ] _[Optional]_ Your README would look nicer if you add a screenshot of your project.

**Review by ElijahTobs — APPROVED** [review]
# STATUS: APPROVED 🎉 🥇 
## Hi Team, 

Good job making the changes required by the previous Code Reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional Changes 👌
_Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better._

- Henceforth, always capitalize your commit messages



_Check for changes marked as Optional in the inline comments below._


_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

_Reviewed by_: __**Elijah Ayandokun**__

**Pull request merged**